test/assertion/commandConstants.ts

```diff
@@ -0,0 +1,7 @@
+export enum commandConstants {
```

Comment:
why do we need to define the constants in tests? We should use the constants from the source code.

Reply:
these are command constants required for command execution needed in NUTs only, we won't need them anywhere else

Reply:
But I believe these constants are already defined in the provardx-plugins-utils

Reply:
And if some these are not defined there, then how are we processing the commands there. I think we should constants for each command we support. The same constant should be used everywhere.